### PR TITLE
Add intraday metrics

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: src/utilities.js [hunk 1]
```diff
@@ -14,8 +14,13 @@ function assertHasData_ (data) {
   if (data === null || data === []) throw new NoDataError_()
 }
 
-function formatDatetimeField_ (field) {
-  return new Date(field).toISOString().slice(0, 10)
+function formatDatetimeField_ (field, datetimeField) {
```
Comment (kalkov) [inline src/utilities.js:17]: Add test for that function.

File: doc/sheet_functions.md [hunk 2]
```diff
@@ -6,6 +6,9 @@
 Active Addresses&quot; refers to the number of unique addresses that
 participated in transactions on a blockchain.</p>
 </dd>
+<dt><a href="#SAN_ACTIVE_ADDRESSES_24H">SAN_ACTIVE_ADDRESSES_24H(projectSlug, from, to)</a> ⇒ <code>Array</code></dt>
```
Comment (kalkov) [inline doc/sheet_functions.md:9]: Do we need `SAN_ACTIVE_ADDRESSES_24H`? Isn't `SAN_ACTIVE_ADDRESSES` with interval argument enough?

Comment (spiderjako) [reply doc/sheet_functions.md:9]: https://github.com/santiment/sanbase2/blob/master/lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json#L157

https://github.com/santiment/sanbase2/blob/master/lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json#L174

They're 2 different metrics, this PR is motivated by the fact that ``SAN_ACTIVE_ADDRESSES_24H`` is the same as ``SAN_ACTIVE_ADDRESSES`` when the interval is ``1d``

Comment (spiderjako) [reply doc/sheet_functions.md:9]: Actually, you're right, going to look into it

File: src/generatedSheetFunctions.js [hunk 3]
```diff
@@ -28,25 +34,14 @@ function SAN_DAILY_AVG_MARKETCAP (projectSlug, from, to, currency) {
 * @returns {Array} of daily closing marketcaps.
 * @customfunction
 */
-function SAN_DAILY_CLOSING_MARKETCAP (projectSlug, from, to, currency) {
```
Comment (kalkov) [inline src/generatedSheetFunctions.js:31]: That should not be removed.

File: test/unit/utilitiesTest.js [hunk 4]
```diff
@@ -31,3 +31,13 @@ describe('beginningOfDaytoEndOfDay_', () => {
     expect(san.beginningOfDaytoEndOfDay_(startOfDay)).to.equalTime(endOfDay)
   })
 })
+
+describe('formatDatetimeField_', () => {
+  it('returns date when the \'field\' argument is date', () => {
```
Comment (kalkov) [inline test/unit/utilitiesTest.js:36]: That's ugly, use double quotes - `"field"`

File: test/unit/utilitiesTest.js [hunk 5]
```diff
@@ -31,3 +31,13 @@ describe('beginningOfDaytoEndOfDay_', () => {
...
+    expect(san.formatDatetimeField_('2020-02-02T00:00:00.000Z', 'date')).to.equal('2020-02-02')
+  })
+
+  it('returns datetime when the \'field\' argument is datetime', () => {
```
Comment (kalkov) [inline test/unit/utilitiesTest.js:40]: That's ugly, use double quotes - "field"

**Review by kalkov — CHANGES_REQUESTED** [review]

**Review by kalkov — APPROVED** [review]

**Pull request merged**